examples/io/plot_read_proj.py

```diff
@@ -0,0 +1,85 @@
+
+"""
+==============================================
+Read and visualize projections (SSP and other)
+==============================================
+
+This example shows how to read and visualize SSP vector
+"""
+# Author: Joan Massich <mailsik@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+
+import mne
+from mne import read_proj
+from mne.io import read_raw_fif
+
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+
+subjects_dir = data_path + '/subjects'
+fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
+ecg_fname = data_path + '/MEG/sample/sample_audvis_ecg-proj.fif'
+
+###############################################################################
+# Load the FIF file and display the recorded projections within the file from
+# the empty room recording.
+raw = read_raw_fif(fname)
+empty_room_proj = raw.info['projs']
+
+# Display the projections stored in `info['projs']` from the raw object
+raw.plot_projs_topomap()
+
+# Display the projections one by one
+fig, axes = plt.subplots(1, len(empty_room_proj))
+for proj, ax in zip(empty_room_proj, axes):
+    proj.plot_topomap(axes=ax)
+
+# Use the function in `mne.viz` to display a list of projections
+assert isinstance(empty_room_proj, list)
+mne.viz.plot_projs_topomap(empty_room_proj)
+
+###############################################################################
+# As shown in the tutorial on how to 
+# :ref:`<sphx_glr_download_auto_tutorials_plot_visualize_raw.py>`
```

Comment:
I don't understand why this does not render properly. The other link I placed does.

![image](https://user-images.githubusercontent.com/7044835/47516734-86961480-d886-11e8-96db-7ebb3f85bd71.png)

Reply:
get rid of the `<>`

Reply:
(and render locally with `make clean; make html_dev-noplot; make show` to check before pushing so you don't have to wait for CircleCI)